autoload/nvimgdb.vim

```diff
@@ -283,6 +287,7 @@ function! s:DefineCommands()
   command! GdbBreakpointToggle call nvimgdb#ToggleBreak()
   command! GdbBreakpointClearAll call nvimgdb#ClearBreak()
   command! GdbRun call nvimgdb#Send("run")
+  command! GdbUntil call nvimgdb#Send("thread until " . line('.'))
```

Comment:
That will work in LLDB, but the command could be easily supported for the whole suite LLDB+GDB+PDB. For that, the backend-specific commands could be defined in the backend dictionary at the beginning of the script, and merely referred here.